config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -3012,6 +3014,2038 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
```

Comment:
Keep the storage version at `v1beta1` until we have a proper migration/conversion strategy?

Reply:
AFAIU we don't need the conversion strategy, because we don't need to migrate any values

Reply:
Taken from: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#writing-reading-and-updating-versioned-customresourcedefinition-objects 
```
if the default strategy value None is specified, the only modifications to the object are changing the apiVersion string and perhaps pruning unknown fields (depending on the configuration).
```
Which is exactly our use case, we just need to prune NL fields

Reply:
Ah ha! I had forgotten the second part apologies, in that case `None` is fine.